packages/zero-cache/src/server/otel-log-sink.ts

```diff
@@ -1,26 +1,24 @@
-import {
-  logs,
-  type Logger,
-  SeverityNumber,
-  type LogRecord,
-  type AnyValueMap,
-} from '@opentelemetry/api-logs';
+import {logs, type Logger, type LogRecord} from '@opentelemetry/api-logs';
 import type {Context, LogLevel, LogSink} from '@rocicorp/logger';
 import {errorOrObject} from './logging.ts';
 import {stringify} from '../types/bigint-json.ts';
-import {startOtel, type OtelEndpoints} from './otel-start.ts';
+import {startOtelAuto} from './otel-start.ts';
 
 export class OtelLogSink implements LogSink {
   readonly #logger: Logger;
 
-  constructor(endpoints: OtelEndpoints) {
+  constructor() {
     // start otel in case it was not started yet
     // this is a no-op if already started
-    startOtel(endpoints);
+    startOtelAuto();
     this.#logger = logs.getLogger('zero-cache');
   }
 
-  log(level: LogLevel, context: Context | undefined, ...args: unknown[]): void {
+  log(
+    level: LogLevel,
+    _context: Context | undefined,
```

Comment:
we need to add the context to the `payload.attributes` otherwise we'll lose all the context that has been set for the logger.

```ts
 if (context) {
    payload.attributes = context as AnyValueMap;
  }
```

Reply:
added context